tests/ui/items_after_test_module.stderr

```diff
@@ -0,0 +1,17 @@
+error: items were found after the testing module
+  --> $DIR/items_after_test_module.rs:13:1
+   |
+LL | / mod tests {
+LL | |     #[test]
+LL | |     fn hi() {}
+LL | | }
+...  |
+LL | |     () => {};
+LL | | }
+   | |_^
```

Comment:
Just a question: is this the intended span for the lint?

Reply:
Yes it is.
Would you want it to start after the test module?

Reply:
This is fine. I was thinking that it might start a line earlier (from `#[cfg(test)]`) but this'd be fine too.